chromium_src/chrome/browser/ui/views/download/download_item_view.cc

```diff
@@ -3,6 +3,23 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
+#define BRAVE_BROWSER_UI_VIEWS_DOWNLOAD_DOWNLOAD_ITEM_VIEW_TRANSPARENT_BTN \
```

Comment:
Seems like this can be done without patches? Something like that:

```
namespace views {

namespace {

class DownloadItemViewButton : public Button {
 public:
  using Button::Button;

  void OnMouseEntered(const ui::MouseEvent& event) override {
    parent()->OnMouseEntered(event);
  }
  void OnMouseExited(const ui::MouseEvent& event) override {
    parent()->OnMouseExited(event);
  }
  void OnViewFocused(View* observed_view) override {
    auto* item = static_cast<DownloadItemView*>(parent());
    item->OnViewFocused(observed_view);
  }
  void OnViewBlurred(View* observed_view) override {
    auto* item = static_cast<DownloadItemView*>(parent());
    item->OnViewBlurred(observed_view);
  }
};

}  // namespace

}  // namespace views

#define Button DownloadItemViewButton

#include "..."

#undef Button
```

Reply:
this worked